src/test/java/org/jabref/gui/importer/ImporterViewModelTest.java

```diff
@@ -0,0 +1,56 @@
+package org.jabref.gui.importer;
+
+import java.nio.file.Path;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+import org.jabref.logic.importer.fileformat.CustomImporter;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ImporterViewModelTest {
+
+    private final CustomImporter importer = mock(CustomImporter.class);
```

Comment:
This whole test does not test any functionality. You are just testing  java's getter and setter or property.
Don't just write tests for the sake of increasing testscoverage. Testing should concencrate on testing logic or implementaition

Reply:
I can of course exclude this test, if you prefer so. I understand that testing logic and implementation is more valueable than increasing coverage. But higher coverage ultimately reaches the same goal. And according to the [comment on issue #6207](https://github.com/JabRef/jabref/issues/6207#issuecomment-717890878) the aim is to reach 100% code coverage.

Reply:
I changed the text, because the knowledge that increasing coverage in an equal way is not as common as I assumed. I mean: If one class has 10% coverage, another has 90%, it is better to work on the class having 10% than trying to increase the coverage from 90% to 100% in the 90% class.